scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -52,7 +52,8 @@ def __init__(self,
                  run_two_suffix = 'test',
                  run_one_description = '',
                  run_two_description = '',
-                 multisubmit = False):
+                 multisubmit = False,
+                 ignore_fieldlist_diffs = False):
```

Comment:
This new constructor argument is the "user interface" to this new change. i.e., this is how you can leverage this feature in system tests that extend SystemTestsCompareTwo.